Source/WebCore/dom/TrustedTypePolicyFactory.idl

```diff
@@ -35,9 +35,9 @@ interface TrustedTypePolicyFactory {
     readonly attribute TrustedHTML emptyHTML;
     readonly attribute TrustedScript emptyScript;
 
-    DOMString? getAttributeType(DOMString tagName, DOMString attribute, optional DOMString elementNamespace = "", optional DOMString attrNamespace = "");
+    DOMString? getAttributeType(DOMString tagName, DOMString attribute, optional [LegacyNullToEmptyString] DOMString elementNamespace = "", optional [LegacyNullToEmptyString] DOMString attrNamespace = "");
```

Comment:
Very surprising that a new function requires this legacy attribute. Do you know why?